src/domains/nodo-common/03_storage_account_nodo_verifyko.tf

```diff
@@ -1,7 +1,3 @@
-data "azurerm_resource_group" "nodo_verifyko_to_datastore_rg" {
-  name = format("%s-verifyko-to-datastore-rg", local.project)
-}
-
 module "nodo_verifyko_storage_account" {
```

Comment:
@aomegax if this storage account requires to be backed up (in prod env), you can add the related configuration alongside delete_retention and versioning_enabled:

```
blob_change_feed_enabled             = var.afm_storage_params.enable_backup
  blob_change_feed_retention_in_days   = var.afm_storage_params.enable_backup ? var.afm_storage_params.backup_retention_days + 1 : null
  blob_container_delete_retention_days = var.afm_storage_params.backup_retention_days
  blob_storage_policy = {
    enable_immutability_policy = false
    blob_restore_policy_days   = var.afm_storage_params.backup_retention_days
  }
  ```
  
  check `module "afm_storage"` for an example